lib/buffer.js

```diff
@@ -637,21 +638,18 @@ Buffer.prototype.toString = function toString(encoding, start, end) {
   }
```

Comment:
Minor nit: can this also be indented correctly?

Reply:
This seems correct? It's indented by exactly two spaces.